apps/src/code-studio/pd/professional_learning_landing/landingPage.module.scss

```diff
@@ -8,6 +9,22 @@ main.wrapper {
   margin-inline: auto;
 }
 
+section {
+  padding: 4rem 1rem 0 1rem !important;
```

Comment:
why does this need to use the `!important` tag?

Reply:
(Unrelated to Bethany's comment above) Thanks for adding this! Can you remove the inline style on line 162 now that this is here?

Reply:
I believe it's because I wasn't able to apply styles to the `HeaderBannerNoImage`'s intrinsic `section` tag so I had to override it in here. If there's a better way to do this I'm definitely open to pushback!